src/main/java/org/prgrms/kdt/dao/entity/voucher/FixedAmountVoucher.java

```diff
@@ -0,0 +1,72 @@
+package org.prgrms.kdt.dao.entity.voucher;
+
+import org.prgrms.kdt.util.ValidatorUtil;
+
+import java.util.Optional;
+import java.util.UUID;
+
+public class FixedAmountVoucher implements Voucher{
+
+    private final UUID voucherId;
+    private final Long discountAmount;
+    private final String voucherType;
+    private UUID ownedCustomerId;
+
+    public FixedAmountVoucher(UUID voucherId, String discountAmount, String voucherType) {
+        validate(discountAmount);
+        this.voucherId = voucherId;
+        this.discountAmount = Long.parseLong(discountAmount);
+        this.voucherType = voucherType;
+    }
+
+
+    @Override
```

Comment:
```suggestion
    }

    @Override
```

개행은 의미전달에 목적으로 사용합니다.